File: src/main/java/com/paragon/api/event/render/world/BlockSetOpaqueEvent.java
package com.paragon.api.event.render.world;

import me.wolfsurge.cerauno.event.CancellableEvent;
import net.minecraft.util.math.BlockPos;

/**
 * @author Surge
 */
public class BlockSetOpaqueEvent extends CancellableEvent {

    private final BlockPos pos;

    public BlockSetOpaqueEvent(BlockPos pos) {
        this.pos = pos;
    }

    /**
     * Gets the pos
     *
     * @return The pos
     */
    public BlockPos getPos() {
        return pos;
    }

}

File: src/main/java/com/paragon/api/event/render/world/BlockSetOpaqueEvent.kt
package com.paragon.api.event.render.world

import me.wolfsurge.cerauno.event.CancellableEvent
import net.minecraft.util.math.BlockPos

/**
 * @author Surge
 */
class BlockSetOpaqueEvent(val pos: BlockPos) : CancellableEvent()